NAMESPACE

```diff
@@ -55,6 +55,7 @@ importFrom(tiltIndicator,emissions_profile)
 importFrom(tiltIndicator,emissions_profile_upstream)
 importFrom(tiltIndicator,jitter_range)
 importFrom(tiltIndicator,nest_levels)
+importFrom(tiltIndicator,percent_noise)
```

Comment:
I think `percent_noise()` is an internal function in tiltIndicator. If you need it here we may need to export it from tiltIndicator or you may copy it here.

Reply:
Are you sure `percent_noise` is an internal function? I think its an external function. Please have a look [here](https://github.com/2DegreesInvesting/tiltIndicator/blob/5ae818397fc64f9cb9e6b5ad3540045025658843/R/percent_noise.R#L7).

Reply:
You're right. Thanks.